hera_cal/smooth_cal.py

```diff
@@ -256,10 +297,13 @@ def time_freq_2D_filter(gains, wgts, freqs, times, freq_scale=10.0, time_scale=1
         method: Algorithm used to smooth calibration solutions. Either 'CLEAN' or 'DPSS':
             'CLEAN': uses the CLEAN algorithm to smooth calibration solutions
             'DPSS': uses discrete prolate spheroidal sequences (DPSS) to filter calibration solutions
-        design_matrix: Matrix of DPSS filters calculated using smooth_cal.dpss_filters. If matrix is not provided,
-            one will be calculated using smooth_cal.dpss_filters and the time and frequency scale. Only used
-            when the method used is 'DPSS'
-        sol_matrix: Solution matrix obtained from performing linear least squares with 2D DPSS vectors
+        design_matrix: Tuple of 2 1D DPSS filters, one for the time axis and one for the frequency axis
```

Comment:
Isn't it a bit confusing to call this the design_matrix? Is there something more precise we can call this pair of filters?

Reply:
Yes, I just switched over to `dpss_vectors` insteads